tests/dockerfile_test.py

```diff
@@ -80,10 +86,12 @@ def test_parse_string_text():
         dockerfile.Command(
             cmd='FROM', sub_cmd=None, json=False, value=('ubuntu:xenial',),
             start_line=1, end_line=1, original='FROM ubuntu:xenial', flags=(),
+            heredocs=(),
         ),
         dockerfile.Command(
             cmd='CMD', sub_cmd=None, json=True, value=('echo', '☃'),
             start_line=2, end_line=2, original='CMD ["echo", "☃"]', flags=(),
+            heredocs=(),
```

Comment:
this indicates an api breaking change

Reply:
Fair point, but this is due to the api breaking change buildkit did by adding the new herdocs structure, so we are consistent with their changes. If there is a way to pass a default value to the go-to-python constructor for the Command, I could do that to avoid this external change, but I have not looked into that yet. If you feel this is important, I can try to add a default empty value. However, the current behavior is consistent with, for example, the flags argument, which is provided an empty value when no flags are used.

Reply:
Right, which is why I replied by pointing out the pros and cons. Pushed a new version that avoids breaking the API.